driver/drivers.go

```diff
@@ -145,6 +146,12 @@ func (d *Drivers) Delete(taskName string) error {
 		return errors.New("task name cannot be empty")
 	}
 
+	driver, ok := d.drivers[taskName]
+
+	if ok {
+		driver.DeleteTask(context.Background())
+	}
```

Comment:
If the map doesn't exist there's nothing to do, we could add a trace log, I don't know if it would be an error.

I think the scenario where the map is empty and Delete is called would be an implementation error, so maybe logging a trace would make sense.

Reply:
Gotcha. That makes sense. I like your idea of adding a log - I think seeing that could be helpful for users.